libs/model/src/policies/CommunityIndexerWorker.policy.ts

```diff
@@ -0,0 +1,151 @@
+import { logger, Policy } from '@hicommonwealth/core';
+import {
+  ClankerToken,
+  CommunityIndexer as CommunityIndexerSchema,
+  EventPairs,
+  events,
+} from '@hicommonwealth/schemas';
+import moment from 'moment';
+import { z } from 'zod';
+import { models } from '../database';
+import { emitEvent } from '../utils';
+import {
+  createCommunityFromClankerToken,
+  paginateClankerTokens,
+} from './utils/community-indexer-utils';
+
+const log = logger(import.meta);
+
+const inputs = {
+  CommunityIndexerTimerTicked: events.CommunityIndexerTimerTicked,
```

Comment:
I am not a fan of having timer-ticked events. Time passing isn't an event. @Rotorsoft do we really need to put cron job logic inside of a policy and then have an event fired on a timer?

Reply:
We’ve already established the convention that timer events should not be emitted to the outbox. Given that, a simple command invoked by cron jobs will be cleaner than having policies react to timer events just to invoke the same command.